R/tdcmm_class.R

```diff
@@ -0,0 +1,87 @@
+# Class description ----
+
+#' `tdcmm` class
+#'
+#' @description The `tdcmm` class is a subclass of a [`tbl_df`][tibble::tbl_df],
```

Comment:
**_Simplified description:_**
@description The `tdcmm` class is a specialized type of data frame that is a subclass of [`tbl_df`][tibble::tbl_df], also known as a "tibble". It is designed to be used with `tidycomm` functions to provide additional information and context to the output tibbles generated by these functions.

This subclass specifically augments output based on statistical tests, by including the relevant model object(s) estimated during the analysis, as well as any performed assumption checks.